src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsNumberAttendeesMapping.java

```diff
@@ -26,12 +27,24 @@ private HearingsNumberAttendeesMapping() {
     }
 
     public static int getNumberOfPhysicalAttendees(@Valid SscsCaseData caseData) {
-        int numberOfAttendees = 0;
 
         if (FACE_TO_FACE != HearingsChannelMapping.getHearingChannel(caseData)) {
-            return numberOfAttendees;
+            return 0;
+        }
+        return getNumberOfFaceToFacePhysicalAttendees(caseData);
+    }
+
+    public static int getNumberOfPhysicalAttendees(@Valid SscsCaseData caseData, ReferenceDataServiceHolder referenceDataServiceHolder) {
+
+        if (FACE_TO_FACE != HearingsChannelMapping.getHearingChannel(caseData, referenceDataServiceHolder)) {
+            return 0;
         }
 
+        return getNumberOfFaceToFacePhysicalAttendees(caseData);
+    }
+
+    private static int getNumberOfFaceToFacePhysicalAttendees(SscsCaseData caseData) {
+        int numberOfAttendees = 0;
```

Comment:
better to initialise numberOfAttendees when you call getNumberOfAppellantAttendees

Reply:
Updated as per the review comments